app/routes/__init__.py

```diff
@@ -209,5 +209,5 @@
 api.add_resource(ProjectUsersHandleInviteView,
                  '/projects/<string:project_id>/users/handle_invite')
 # system status
-api.add_resource(SystemStatusView, '/system_status')
```

Comment:
Hello @mirembe-mariam . Nice job on this . In this case it is also worth removing the corresponding controller `SystemStatusView` from `app/controllers/system_status/SystemStatusView` to avoid redundant code ...

Reply:
To add more on @LanternNassi comment, you need to delete these two files `app/controllers/system_status.py` and `app/controllers/statuses.py`

Reply:
Hello @Mubangizi  deleting the file `app/controllers/system_status.py` will remove a crucial endpoint used for loading summary data such as databases , apps , projects `/system_summary` on the frontend . I think she should just refactor the required controller and may be delete the `app/controllers/statuses.py` file only .

Reply:
Oh yeah thats a good catch @LanternNassi, Mariam please only remove `SystemStatusView` class in `app/controllers/system_status.py` file. My bad I thought there was only one controller in there